### PR TITLE
chore: update packages

### PR DESCRIPTION
**Pull request by ivanoats**
Just keeping packages up to date. Tests pass locally.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import logo from './logo.svg';
 
 const textFontSizes = [16, 18, 24, 30];
 
-const App = (): JSX.Element => {
+function App(): JSX.Element {
```
Comment (ivanoats) [inline src/App.tsx:11]: auto linting fix

File: src/components/ThemeToggleButton.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const RoundButton = styled(IconButton, transientOptions)`
   }
 `;
 
-const ThemeToggleButton = (props: ThemeToggleButtonProps): JSX.Element => {
+function ThemeToggleButton(props: ThemeToggleButtonProps): JSX.Element {
```
Comment (ivanoats) [inline src/components/ThemeToggleButton.tsx:22]: auto linting fix

**Comment by ivanoats** [comment]
Looks like deploy is failing on netlify's node version `10:12:44 AM: error @typescript-eslint/eslint-plugin@5.12.1: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "12.18.0"`

**Comment by The24thDS** [comment]
@ivanoats Thank you for your contribution! I appreciate it! I just had to rename the `.nvmrvc` file that you made to `.nvmrc` for Netlify to pick it up correctly. Hope you don't mind! : )

**Comment by ivanoats** [comment]
> @ivanoats Thank you for your contribution! I appreciate it! I just had to rename the `.nvmrvc` file that you made to `.nvmrc` for Netlify to pick it up correctly. Hope you don't mind! : )

Haha, thanks so much. My eyes are not always reliable 👀

**Pull request merged**